python/desc/sims/GCRCatSimInterface/DatabaseEmulator.py

```diff
@@ -517,7 +520,73 @@ class SNFileDBObject(fileDBObject):
                ('Tx0', 'x0_in'),
                ('Tx1', 'x1_in'),
                ('Tc', 'c_in'),
-               ('id', 'snid_in'),
+               ('id', 'snid_in', str, 20),
```

Comment:
Will this be the object id in the instance catalog?   So for the example files Rahul provided, they would be object ids like `MS_564_7` indicating SNe 7 in the main survey region in healpixel 564?

Reply:
Yes.  I should probably bump the character limit up, just in case.